cns/networkcontainers/networkcontainers_linux.go

```diff
@@ -3,20 +3,180 @@
 
 package networkcontainers
 
-import "github.com/Azure/azure-container-networking/cns"
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"go/types"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/containernetworking/cni/libcni"
```

Comment:
vendor this

Reply:
As discussed offline, lets do it as part of another checkin ( with full validation)